### DISCUSSION TITLE
Add ability to access EC engine from a plugin

### DISCUSSION THREAD

**Discussion by Adammatthiesen** [expressive-code/expressive-code#379] [Feature Requests]
Would be useful for plugins like [`expressive-code-twoslash`](https://twoslash.matthiesen.dev)

see Astro discord for context https://discord.com/channels/830184174198718474/1070481941863878697/1310966229258997810

**Comment by delucis** [comment]
Hello! We are moving feature requests and support questions to discussions and using issues only for bugs to help better track those going forward. See you in the [Discussions](https://github.com/expressive-code/expressive-code/discussions) tab!
